### DISCUSSION TITLE
Error on ".well-known" URLs

### DISCUSSION THREAD

**Discussion by nikkrr** [nextcloud/all-in-one#4635] [Questions] [answered]
I moved from nextcloud to nextcloud-aio today and so far everything is working fine. However, I have a single error that I can't get rid of:
**_- Your web server is not properly set up to resolve `.well-known` URLs, failed on: `/.well-known/caldav` For more details see the [documentation ↗](https://docs.nextcloud.com/server/29/go.php?to=admin-setup-well-known-URL)._**

I don't know exactly what I'm doing wrong, because I think everything is actually stored in the .htaccess file of Nextcloud to ensure a correct resolution of the URL.

My host system is Unraid (version: 6.12.10) and I use NGINX Proxy Manager which is connected in front of Nextcloud (I have attached pictures). NPM is in a Docker container, and uses the host network of Unraid. In the native Nextcloud container, the error did not occur.

<img width="1160" alt="Bildschirmfoto 2024-05-11 um 14 39 03" src="https://github.com/nextcloud/all-in-one/assets/63823657/3b66e97b-a6d6-46cf-ae8f-73d3b434a859">
<img width="487" alt="Bildschirmfoto 2024-05-11 um 14 39 16" src="https://github.com/nextcloud/all-in-one/assets/63823657/d9bae71c-a074-432e-807c-da669886d43d">
<img width="487" alt="Bildschirmfoto 2024-05-11 um 14 39 26" src="https://github.com/nextcloud/all-in-one/assets/63823657/fbc53b83-edb9-4546-8419-22f2e5be270b">
<img width="492" alt="Bildschirmfoto 2024-05-11 um 14 39 33" src="https://github.com/nextcloud/all-in-one/assets/63823657/289f696d-6d70-4df1-a99e-3e46cdc9cb8f">
<img width="491" alt="Bildschirmfoto 2024-05-11 um 14 39 39" src="https://github.com/nextcloud/all-in-one/assets/63823657/81a3dc51-921e-4a97-a891-767ea847b1ca">
<img width="1502" alt="Bildschirmfoto 2024-05-11 um 14 45 44" src="https://github.com/nextcloud/all-in-one/assets/63823657/2ba91f83-6c36-4e93-baf3-e5b68e445db3">

**Comment by szaimen** [comment, marked as answer]
Hi, this is a bug in Nextcloud 29.0.0 and should hopefully get addressed with 29.0.1

**Reply by ericmail84** [reply]
Still present in 29.0.1 as far as I can tell.

**Reply by szaimen** [reply]
It will ne fixed in the next AIO update: https://github.com/nextcloud/all-in-one/pull/4651

**Reply by ericmail84** [reply]
Sorry, I misunderstood.
